cuda2gcn/src/math.cl

```diff
@@ -0,0 +1,279 @@
+/*===--------------------------------------------------------------------------
+ *                   ROCm Device Libraries
+ *
+ * This file is distributed under the University of Illinois Open Source
+ * License. See LICENSE.TXT for details.
+ *===------------------------------------------------------------------------*/
+
+#include "ocml.h"
+
+#define ATTR __attribute__((always_inline const))
+
+#define FUNC1D(root, x) \
```

Comment:
I don't really see the point of adding the parameter name to the macro argument.